app/policies/users/application_policy.rb

```diff
@@ -16,4 +16,11 @@ def grant_signin_permission?
   end
 
   alias_method :remove_signin_permission?, :grant_signin_permission?
+
+  def edit_permissions?
+    return false unless Pundit.policy(current_user, user).edit?
+    return true if current_user.govuk_admin?
+
+    current_user.publishing_manager? && current_user.has_access_to?(application) && application.has_delegatable_non_signin_permissions?
```

Comment:
Update this (and anything else) to be what it currently is, then make the changes in the next branch